<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>891</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hymodynamics Platform</string>
  </property>
  <property name="windowIcon">
   <iconset theme="Hemodynamics"/>
  </property>
  <property name="iconSize">
   <size>
    <width>80</width>
    <height>40</height>
   </size>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Nastran"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuViewport">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Viewport</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAnimate">
    <property name="title">
     <string>Animate</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuModel"/>
   <addaction name="menuViewport"/>
   <addaction name="menuHelp"/>
   <addaction name="menuPlot"/>
   <addaction name="menuAnimate"/>
   <addaction name="menuReport"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDockWidget" name="ConsoleDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>623</width>
     <height>746</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_73">
     <item>
      <widget class="QFrame" name="QFrame_Menu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="QLabel_Modules">
          <property name="text">
           <string>Fill Mask with Values</string>
          </property>
          <property name="buddy">
           <cstring>QComboBox_Modules</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="QComboBox_Modules">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>VisualizationParaview</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mask Generation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Save VTU</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centerline To Cut Plane</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VisualizationPyVista</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VTU Data Extraction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>One Click CFD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Filter Mask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centerline Extraction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Update Centerline JSON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Centerline Xsection</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lumen Correction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Convert Data Type</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Xsection Original Volume</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stack Segmentation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Segmentation Extension</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mesh Cut</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mask to STL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Batch Pre Processing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Batch Coronary FSI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Area and Flow Calculation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blood Pressure And Heart Rate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Periluminal Segmentation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tissue Segmentation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mask Dilation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mask Shrinking</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mask Coordinate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TissueElemAssign</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CFD Pre-Process</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>StackSTL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MeshFillGap</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mesh</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gnerate InputFile</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FaceElemSplit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OneClicked</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Solver</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DataOutput</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NodeMatching</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PostProcessing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CFD Post-processing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Map Results</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FFR Calculation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>View Results 3D</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MPR CPR Generation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Image Display</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Radial Unfolding</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Straight CPR Point Stats</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Batch Slice Radiomics</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Extract 2D Slices</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Segment Info Extract</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_modules">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>804</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="InitDispBtn_VTKQT">
          <property name="text">
           <string>Initial Display</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QStackedWidget" name="QStackedWidget_Module">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="MessageDockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>176</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>260</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_31">
     <item>
      <widget class="QGroupBox" name="QGroupBox_Info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_75">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTabWidget" name="QTabWidget_Hint">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="Hint">
            <attribute name="title">
             <string>Hint</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_72">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_Hint">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="plainText">
                <string>Title: Arial 14 HCenter autofill background</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTabWidget" name="QTabWidget_Message">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="Message">
            <attribute name="title">
             <string>Message</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_Message">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="plainText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Log">
            <attribute name="title">
             <string>Log</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_Log">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionImport_Nastran">
   <property name="text">
    <string>Import Nastran</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QComboBox_Modules</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QStackedWidget_Module</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
